_posts/2020-11-31-homebrew-2.6.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 2.6.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
+
+Major changes and deprecations since 2.5.0:
+
+- [macOS Big Sur is supported (and High Sierra unsupported)](https://github.com/Homebrew/brew/pull/9054)
+- [`depends_on :java`, `brew switch`, `brew diy` and various other APIs have been deprecated](https://github.com/Homebrew/brew/pull/9209)
+- [All `brew cask` commands have been deprecated in favour of `brew` commands (with `--cask`) when necessary](https://github.com/Homebrew/brew/pull/8899)
```

Comment:
I've noticed a few people reporting issues related to a change in `brew list` functionality that was deprecated in homebrew/brew#8851 (released in 2.5.7) and I believe it's intended to match the deprecation of `brew cask` commands. However, I believe the `odeprecated` calls from homebrew/brew#8851 were included in the conversions to `odisabled` in homebrew/brew#9209. I'm going to open a PR to narrowly un-`odisable` the changes from homebrew/brew#8851

Reply:
https://github.com/Homebrew/brew/pull/9357

Reply:
Merged!